server/core/auth.js

```diff
@@ -125,7 +125,16 @@ module.exports = {
           return next()
         }
       }
-
+      // JWT is valid but user is NOT active
+      if (user) {
+        const dbUser = await WIKI.models.users.query().findOne({
+          email: user.email,
+          providerKey: 'local'
```

Comment:
Why is local hardcoded? Should be using whatever provider the user currently has.

Reply:
I updated the PR to use the provider on the user.